mesop/components/code/code.py

```diff
@@ -0,0 +1,10 @@
+from mesop.component_helpers import component
+from mesop.components.markdown.markdown import markdown
+
+
+@component
+def code(code: str = "", *, language: str = "python"):
+  """
+  Creates a code component which displays code with syntax highlighting.
+  """
+  markdown("``` " + language + "\n" + code + "\n" + "```")
```

Comment:
Seems like it could be possible for someone to enter something that could break this code block if the user was accepting user input for the code block params.

For example if I did a language of ```python```.

But seems like it should be ok since it would still go through the markdown component, I think some kind of injection attack would not work. The code block would just not render correctly.

Reply:
Good point, I agree that this isn't a security issue since we're relying on Angular's DOM sanitization https://angular.io/guide/security#sanitization-and-security-contexts, which is done in the markdown component.